### PR TITLE
Feature/claudia 5589 resolve unit test execution

### PR DESCRIPTION
**Pull request by flopezag**
#### Reviewers

 @chemaper  , @jframos 
#### Description
- Passing PEP8 coding format.
- Changing configuration of Sonar project properties.
- Resolve the execution of unit test with nosetests (execution from root directory).
- Adding configuration of coverage tool.
- Changing in .gitignore.
- Resolve a problem in acceptance tests when we try to compare metadata attributes with no values returned.
#### Testing
- Locally Aceptance tests, in Jenkins the execution and in Sonar the publication of data.

### CODE REVIEW

**Comment by jframos** [comment]
What about `scripts/run_unit_tests.sh` ?

**Comment by flopezag** [comment]
From my point of view it is something that could be deleted, but I need your confirmation before doing it, @chemaper @jframos ?

**Comment by jframos** [comment]
Perhaps we can move the script to the main folder and rename it to `build.py` or something like this, with all building activities, being the nosetests execution one part of that script. Jenkins could only call that script from the task.

**Comment by flopezag** [comment]
The script is doing only the nosetests execution, from my point of view we do not it to anything else.

**Comment by jframos** [comment]
NTC

File: tests/unit/test_glancesync.py [hunk 1]
```diff
@@ -349,7 +349,9 @@ class TestGlanceSync_Sync(unittest.TestCase):
     """
 
     def config(self):
-        self.path_test = '../resources/alreadysync'
+        path = os.path.abspath(os.curdir)
+        print 'Resolving problem in test_glancesync.py'
+        self.path_test = path + '/tests/resources/alreadysync'
         self.regions = ['Valladolid', 'master:Burgos', 'other:Madrid']
```
Comment (chemaper) [inline tests/unit/test_glancesync.py:354]: I suppose the print line is a forgotten trace. Isn't it?

Comment (flopezag) [inline tests/unit/test_glancesync.py:354]: Yes you are right, I will change it.

**Comment by jframos** [comment]
LGTM

**Comment by jframos** [comment]
Please, update documentation: _Testing/Unit tests_

**Comment by jframos** [comment]
LGTM

**Comment by chemaper** [comment]
LGTM

**Pull request merged**